src/tools/tests/goldens/generated-schemas.kt

```diff
@@ -185,7 +160,47 @@ class Gold_Data() : Entity<Gold_Data>() {
     }
 }
 
+class Gold_Data_Spec() : EntitySpec<Gold_Data> {
+
+    override fun create() = Gold_Data()
+
+    override fun decode(encoded: ByteArray): Gold_Data? {
+        if (encoded.isEmpty()) return null
+
+        val decoder = StringDecoder(encoded)
+        decoder.validate("|")
+        val entity = create().apply {
+            internalId = decoder.decodeText()
+            for (_i in 0 until 5) {
+                if (decoder.done()) break
+                val name = decoder.upTo(':').utf8ToString()
+                when (name) {
+                    "num" -> {
+                        decoder.validate("N")
+                        this.num = decoder.decodeNum()
+                    }
+                    "txt" -> {
+                        decoder.validate("T")
+                        this.txt = decoder.decodeText()
+                    }
+                    "lnk" -> {
+                        decoder.validate("U")
+                        this.lnk = decoder.decodeText()
+                    }
+                    "flg" -> {
+                        decoder.validate("B")
+                        this.flg = decoder.decodeBool()
+                    }
+                }
+                decoder.validate("|")
+            }
+        }
+        return entity
```

Comment:
same, it could be inlined..

Reply:
Done. (this file is generated -- I updated the generator)